<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P2CogView</class>
 <widget class="QWidget" name="P2CogView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>361</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans Mono</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>COG view</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="frame_CTx">
     <property name="toolTip">
      <string>Count match values for CT1, CT2, and CT3</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_CT1">
        <property name="text">
         <string>CT1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_CT1"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lbl_CT2">
        <property name="text">
         <string>CT2:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="le_CT2"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="lbl_CT3">
        <property name="text">
         <string>CT3:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="le_CT3"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="group_PC">
     <property name="toolTip">
      <string>Program counter (PC), intruction register, C flag, Z flag</string>
     </property>
     <property name="title">
      <string>COG ID #0</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_PC">
        <property name="text">
         <string>PC:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_PC"/>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="cb_Z">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="cb_C">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_IR"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QFrame" name="frame_LOCK">
     <property name="toolTip">
      <string>LOCK mode / edge / number values</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>LOCK:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="le_LOCK_num"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_LOCK_mode"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_LOCK_edge"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame_FLAGS">
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>COG status flags</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QCheckBox" name="cb_CT2">
        <property name="text">
         <string>CT2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cb_CT1">
        <property name="text">
         <string>CT1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QCheckBox" name="cb_SE2">
        <property name="text">
         <string>SE2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="cb_XFI">
        <property name="text">
         <string>XFI</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cb_PAT">
        <property name="text">
         <string>PAT</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QCheckBox" name="cb_XRL">
        <property name="text">
         <string>XRL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QCheckBox" name="cb_SE4">
        <property name="text">
         <string>SE4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="cb_XMT">
        <property name="text">
         <string>XMT</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="cb_XRO">
        <property name="text">
         <string>XRO</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QCheckBox" name="cb_ATN">
        <property name="text">
         <string>ATN</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="cb_SE1">
        <property name="text">
         <string>SE1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QCheckBox" name="cb_QMT">
        <property name="text">
         <string>QMT</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QCheckBox" name="cb_SE3">
        <property name="text">
         <string>SE3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="cb_CT3">
        <property name="text">
         <string>CT3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cb_INT">
        <property name="text">
         <string>INT</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cb_FBW">
        <property name="text">
         <string>FBW</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QFrame" name="frame_PAT">
     <property name="toolTip">
      <string>PAT mode, mask, match values</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_PAT">
        <property name="text">
         <string>PAT:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_PAT_mode"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_PAT_mask"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="le_PAT_match"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QFrame" name="frame_PIN">
     <property name="toolTip">
      <string>PIN mode, edge, and number values</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_PIN">
        <property name="text">
         <string>PIN:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_PIN_mode"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_PIN_edge"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="le_PIN_num"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tvDasm">
     <property name="toolTip">
      <string>Disassembled instruction</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_PC</tabstop>
  <tabstop>le_IR</tabstop>
  <tabstop>cb_C</tabstop>
  <tabstop>cb_Z</tabstop>
  <tabstop>tvDasm</tabstop>
  <tabstop>cb_INT</tabstop>
  <tabstop>cb_CT1</tabstop>
  <tabstop>cb_CT2</tabstop>
  <tabstop>cb_CT3</tabstop>
  <tabstop>cb_SE1</tabstop>
  <tabstop>cb_SE2</tabstop>
  <tabstop>cb_SE3</tabstop>
  <tabstop>cb_SE4</tabstop>
  <tabstop>cb_PAT</tabstop>
  <tabstop>cb_FBW</tabstop>
  <tabstop>cb_XMT</tabstop>
  <tabstop>cb_XFI</tabstop>
  <tabstop>cb_XRO</tabstop>
  <tabstop>cb_XRL</tabstop>
  <tabstop>cb_ATN</tabstop>
  <tabstop>cb_QMT</tabstop>
  <tabstop>le_CT1</tabstop>
  <tabstop>le_CT2</tabstop>
  <tabstop>le_CT3</tabstop>
  <tabstop>le_PAT_mode</tabstop>
  <tabstop>le_PAT_mask</tabstop>
  <tabstop>le_PAT_match</tabstop>
  <tabstop>le_PIN_mode</tabstop>
  <tabstop>le_PIN_edge</tabstop>
  <tabstop>le_PIN_num</tabstop>
  <tabstop>le_LOCK_mode</tabstop>
  <tabstop>le_LOCK_edge</tabstop>
  <tabstop>le_LOCK_num</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
